scripts/mac_refresh.py

```
@@ -0,0 +1,92 @@
+import urllib.request
+import re
+from pathlib import Path
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
+from cryptography.hazmat.primitives import hashes
+
+def get_mac_slack_key():
+    cmd = ["security", "find-generic-password", "-s", "Slack Safe Storage", "-w"]
+    try:
+        return subprocess.check_output(cmd, stderr=subprocess.DEVNULL).strip()
```

Ruff (0.15.15)

[error] Undefined name `subprocess`

(F821)